pyramid_oereb/contrib/print_proxy/mapfish_print.py

```diff
@@ -94,8 +94,13 @@ def __call__(self, value, system):
             cmd = ['pdftk', main.name]
             temp_files = [main]
             for url in pdf_to_join:
-                tmp_file = tempfile.NamedTemporaryFile(suffix='.pdf')
                 result = requests.get(url)
+                content_type = result.headers.get('content-type')
+                log.debug("document url: " + url + " => content_type: " + content_type)
+                if content_type != 'application/pdf':
+                    log.warn("skipped document inclusion (url: " + url + " ) because content_type: " + content_type)
```

Comment:
better with format =>
```
"Skipped document inclusion (url: '{}') because content_type: '{}'".format(url, content_type)
```

I also add quot and a capital :-)